modules/integration/tests-integration/tests-amqp/src/test/java/org/wso2/mb/integration/tests/amqp/functional/DurableTopicServerRestartTestCase.java

```diff
@@ -85,25 +86,24 @@ public void performDurablePublishRestartServerTestCase()
         AndesClient initialConsumerClient = new AndesClient(consumerConfig, true);
         initialConsumerClient.startClient();
 
+        // Wait till consumer starts listening
         AndesClientUtils.sleepForInterval(5000L);
 
         // Stopping the subscription
         initialConsumerClient.stopClient();
 
+        // Wait till consumer stops
         AndesClientUtils.sleepForInterval(5000L);
 
         // Creating the publisher and publishing
         AndesClient publisherClient = new AndesClient(publisherConfig, true);
         publisherClient.startClient();
 
-        AndesClientUtils.sleepForInterval(15000L);
+        AndesClientUtils.sleepForInterval(5000L);
 
```

Comment:
Yes, Will use constants.
Please see fix in https://github.com/wso2/product-mb/pull/57